public/css/styles.css

```diff
@@ -292,23 +373,24 @@ header {
 }
 
 .vote-selected {
-  border-width: 5px;
-  border-style: solid;
-  border-color: VAR(--accent-color);
+  border-width: 5px !important;
```

Comment:
Why are the !important needed Since this is the most specific class styling would that mean an id styling is overwriting these specifications?